spec/xclarity_client_storage_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'spec_helper'
+
+describe XClarityClient do
+  before :all do
+    WebMock.allow_net_connect! #-- Uncomment this line if you're testing with a external mock.
+  end
+
+  let(:conf) do
+    XClarityClient::Configuration.new(
+      username:   ENV['LXCA_USERNAME'],
+      password:   ENV['LXCA_PASSWORD'],
+      host:       ENV['LXCA_HOST'],
+      port:       ENV['LXCA_PORT'],
+      auth_type:  ENV['LXCA_AUTH_TYPE'],
+      verify_ssl: ENV['LXCA_VERIFY_SSL']
+    )
+  end
+
+  let(:client) { XClarityClient::Client.new conf }
+
+  it 'has a version number' do
+    expect(XClarityClient::VERSION).not_to be nil
+  end
+
+  context 'with discover_storages' do
+    before do
+      @storages = client.discover_storages
+    end
+
+    describe 'GET /storage' do
+      it 'should respond with an array' do
+        expect(@storages.class).to eq(Array)
+        expect(@storages.any?).to eq(true)
+      end
+
+      it 'correct response values' do
+        @storages.each do |storage|
+          expect(storage.name).to eq('S3200-1')
+          expect(storage.type).to eq('Lenovo Storage')
+          expect(storage.accessState).to eq('Online')
+          expect(storage.driveBays).to eq(12)
+          expect(storage.enclosureCount).to eq(1)
+          expect(storage.canisterSlots).to eq(2)
+        end
+      end
+    end
+  end
+
+  context 'with fetch_storages' do
+    before do
+      @include_attributes = %w[name type accessState driveBays enclosureCount canisterSlots]
+      @exclude_attributes = %w[type]
+      @uuids = client.discover_storages.select { |storage| storage.uuid }
+    end
+
+    it 'missing attributes should be nil' do
+      storages = client.fetch_storages(@uuids, @include_attributes)
+      storages.map do |storage|
```

Comment:
__Suggestion:__
In this test you may test if the `storages.count` is equal at `@uuids.count`.

__Required:__
Like I said before, to check the right values you can access the storage node by its index in array